File: DigiMox/app/src/main/java/com/digimox/adapters/DMDetailParentPagerAdapter.java
package com.digimox.adapters;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentPagerAdapter;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by Deepesh on 10-Dec-15.
 */
public class DMDetailParentPagerAdapter extends FragmentPagerAdapter {
    private List<Fragment> mFragmentList;

    public DMDetailParentPagerAdapter(FragmentManager fm) {
        super(fm);
        mFragmentList = new ArrayList<Fragment>();
    }

    public void addFragment(Fragment fragment) {
        mFragmentList.add(fragment);
    }

    @Override
    public int getCount() {
        return mFragmentList.size();
    }

    @Override
    public Fragment getItem(int position) {
        return mFragmentList.get(position);
    }

    @Override
    public CharSequence getPageTitle(int position) {
        return "";
    }
}

File: DigiMox/app/src/main/java/com/digimox/adapters/DMDetailParentPagerAdapter.kt
package com.digimox.adapters

import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentPagerAdapter

import java.util.ArrayList

/**
 * Created by Deepesh on 10-Dec-15.
 */
class DMDetailParentPagerAdapter(fm: FragmentManager) : FragmentPagerAdapter(fm) {
    private val mFragmentList: MutableList<Fragment>

    init {
        mFragmentList = ArrayList()
    }

    fun addFragment(fragment: Fragment) {
        mFragmentList.add(fragment)
    }

    override fun getCount(): Int {
        return mFragmentList.size
    }

    override fun getItem(position: Int): Fragment {
        return mFragmentList[position]
    }

    override fun getPageTitle(position: Int): CharSequence? {
        return ""
    }
}
